js/components/request_summary_contact_section.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function RequestSummaryContactSection({ formData }) {
+  const data = formData.requester_contact;
+  return (
+    <div className="request-summary_section">
+      <h5>Name</h5>
+      <p>{[data.first_name, data.last_name].join(' ')}</p>
+      { data.mailing_address_line_1 &&
+        <div>
+          <h5>Mailing address</h5>
+          <address>
+            <p>{ data.mailing_address_line_1 }</p>
+            <p>{ data.mailing_address_line_2}</p>
+            <p>{ data.city }, { data.state_province } { data.zip_postal_code }</p>
```

Comment:
these fields should be: `address_line1`, `address_line2`, `address_city`, `address_state_province`, `address_zip_postal_code`.

Let's also include country below. Refer to https://github.com/18F/beta.foia.gov/blob/develop/js/util/request_form/form_sections.js#L11 for the new field names.